### PR TITLE
GraphNG: Fix issues with plugins not retrieving plot instance

### PR DESCRIPTION
**Pull request by dprokop**
Fixes issue when plugins did not initialize because `isPlotReady` flag was removed from the context.

### CODE REVIEW

File: packages/grafana-ui/src/components/uPlot/Plot.tsx [hunk 1]
```diff
@@ -72,13 +69,19 @@ export const UPlotChart: React.FC<PlotProps> = props => {
...
+    return buildPlotContext(canvasRef, props.data, registerPlugin, getPlotInstance);
   }, [plotInstance, canvasRef, props.data, registerPlugin, getPlotInstance]);
 
   return (
     <PlotContext.Provider value={plotCtx}>
-      <div ref={plotCtx.canvasRef} data-testid="uplot-main-div" />
-      {props.children}
+      <div
+        className={css`
+          position: relative;
```
Comment (torkelo) [inline packages/grafana-ui/src/components/uPlot/Plot.tsx:79]: really dislike these inline emotion stiles (I do them as well sometimes), which hurts the readability of the jxs markup, wonder if should add a utility emotion classNames for some common flex & css props.

**Review by torkelo — APPROVED** [review]
Looks good!

File: packages/grafana-ui/src/components/uPlot/context.ts [hunk 2]
```diff
@@ -127,27 +126,25 @@ export const usePlotData = (): PlotDataAPI => {
...
+      height: getPlotInstance()!.height,
       plot: {
-        width: getPlotInstance().bbox.width / window.devicePixelRatio,
-        height: getPlotInstance().bbox.height / window.devicePixelRatio,
-        top: getPlotInstance().bbox.top / window.devicePixelRatio,
-        left: getPlotInstance().bbox.left / window.devicePixelRatio,
+        width: getPlotInstance()!.bbox.width / window.devicePixelRatio,
+        height: getPlotInstance()!.bbox.height / window.devicePixelRatio,
+        top: getPlotInstance()!.bbox.top / window.devicePixelRatio,
+        left: getPlotInstance()!.bbox.left / window.devicePixelRatio,
```
Comment (leeoniya) [inline packages/grafana-ui/src/components/uPlot/context.ts:147]: could be a bit more terse by aliasing above the return:

```js
const plot = getPlotInstance()!,
      bbox = plot.bbox,
      pxRatio = window.devicePixelRatio;
```

Comment (torkelo) [reply packages/grafana-ui/src/components/uPlot/context.ts:147]: yea, nice to get rid of all the calls to getPlotInstance()!   :)

Comment (dprokop) [reply packages/grafana-ui/src/components/uPlot/context.ts:147]: 👍

**Pull request merged**